core/src/martian/encoders.cljc

```diff
@@ -37,15 +38,15 @@
   ([key-fn]
    (merge
     #?(:clj
-       {"application/transit+msgpack" {:encode #(transit-encode % :msgpack)
+       {"application/transit+json"    {:encode #(transit-encode % :json)
```

Comment:
The order is important here (although it looks accidental because of not using an ordered map for each of these cases), as it specifies an order of preference of encoders. If bb doesn't support this one, can we just have an extra `:bb` dispatch which doesn't include it?